### PR TITLE
Add v1.13.0 release blog post

### PR DESCRIPTION
**Pull request by pmuens**
**Todos:**

- [x] Update contributors list

### CODE REVIEW

File: posts/2017-05-10-serverless-v1.13.0.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: New in Serverless v1.13 - Use values of async functions in your Serverless variables
```
Comment (DavidWells) [inline posts/2017-05-10-serverless-v1.13.0.md]: What about something like: 

"Serverless v1.13 opens up world of configuration possibilities with new async variable support"

File: posts/2017-05-10-serverless-v1.13.0.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Today we're proud to announce the release of the Serverless Framework v1.13!
+
+Let's look into the new features and enhancements Serverless v1.13 introduces.
+
+## Highlights of 1.13.0
+
+**Note:** You can find a complete list of all the updates in the [changelog](https://github.com/serverless/serverless/blob/master/CHANGELOG.md).
+
+### Async value support for Serverless variables
+
```
Comment (DavidWells) [inline posts/2017-05-10-serverless-v1.13.0.md]: We should add list of config possibilities here:

- Connect variables to your teams secret management solution
- Fetch secrets from a remote database
- etc.

Comment (pmuens) [inline posts/2017-05-10-serverless-v1.13.0.md]: +1000000

File: posts/2017-05-10-serverless-v1.13.0.md (hunk 1 above)
Comment (pmuens) [inline posts/2017-05-10-serverless-v1.13.0.md]: Nice one! 👍

File: posts/2017-05-10-serverless-v1.13.0.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
+---
+title: New in Serverless v1.13 - v1.13 opens up world of configuration possibilities with new async variable support
```
Comment (DavidWells) [inline posts/2017-05-10-serverless-v1.13.0.md]: Title too long and repeats =)

How about:

"Serverless v1.13 - Async variable support added for easier configuration & secret management"

Comment (DavidWells) [inline posts/2017-05-10-serverless-v1.13.0.md]: "Async variable support added for easier configuration & secret management - Serverless v1.13"
 
or "Async variable support added in Serverless v1.13 for better configuration & secret management"

or "Async variable support added in Serverless v1.13 for enhanced configuration & secret management" <----- My faves

Comment (pmuens) [inline posts/2017-05-10-serverless-v1.13.0.md]: Thanks for the suggestions! Updating...

**Pull request merged**